daemon/logger/logentries/logentries.go

```diff
@@ -38,32 +42,42 @@ func init() {
 func New(info logger.Info) (logger.Logger, error) {
 	logrus.WithField("container", info.ContainerID).
 		WithField("token", info.Config[token]).
+		WithField("line-only", info.Config[lineonly]).
 		Debug("logging driver logentries configured")
 
 	log, err := le_go.Connect(info.Config[token])
 	if err != nil {
 		return nil, err
 	}
+	var lineOnly bool
+	if lineOnly, err = strconv.ParseBool(info.Config[lineonly]); err != nil {
+		return nil, errors.Wrap(err, "error parsing lineonly option")
+	}
 	return &logentries{
 		containerID:   info.ContainerID,
 		containerName: info.ContainerName,
 		writer:        log,
+		lineOnly:      lineOnly,
 	}, nil
 }
 
 func (f *logentries) Log(msg *logger.Message) error {
-	data := map[string]string{
-		"container_id":   f.containerID,
-		"container_name": f.containerName,
-		"source":         msg.Source,
-		"log":            string(msg.Line),
-	}
-	for k, v := range f.extra {
-		data[k] = v
+	if !f.lineOnly {
+		data := map[string]string{
+			"container_id":   f.containerID,
+			"container_name": f.containerName,
+			"source":         msg.Source,
+			"log":            string(msg.Line),
+		}
+		for k, v := range f.extra {
+			data[k] = v
+		}
+		logger.PutMessage(msg)
```

Comment:
That seems like it would work since we aren't actually modifying the timestamp itself at any point.